src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
>```rust
impl RgbSpace for Srgb {
    type Primaries = Srgb;
    type WhitePoint = D65;
}
```

That's still good old sRGB. What I meant was the sRGB primaries with any other white point. No adaption, just a different combination.